packages/apps/patterns/react-appkit/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,71 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import React from 'react';
+
+import { DocumentBase, id, type } from '@dxos/echo-schema';
+import { Group } from '@dxos/react-components';
+
+import { Task, TaskList } from '../../proto';
+
+export type Card<T = any> = {
```

Comment:
```suggestion
export type Card<T = DocumentBase> = {
```

@dmaretskyi I think ideally it wouldn't use `any` however I had issues reading any properties out of `DocumentBase` because typescript gets angry (e.g., https://github.com/dxos/dxos/pull/2401/files#diff-ccfe78b5591675c1b7ef0aaca22664f18ec216651705ebf1a87abb8074432f73R22)